docs/UserGuide.md

```diff
@@ -1,186 +1,205 @@
-> The index **must be a positive integer** 1, 2, 3, ...
+### 3.4 Updating a field of a task/event in the active list : `edit`
 
-Examples:
+> User can edit the fields of the tasks and events in the active list by using their indexes specified in the most recent list. The steps are therefore as follows:
+
+1. Use `list` command
+2. Using the index, call `edit INDEX etc.`
+
+**Updating the fields**:
+
+Format: `edit INDEX FIELD_NAME NEW_VALUE`
+
+Examples: 
+
+`edit 1 priority 3`, assuming that the item with index 1 is a task.
 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br>
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+### 3.5 Mark as `done` or dimiss tasks and events: `done` and `dis`
+> Just like the `edit` command, the user makes use of the index provided by the `list`-type commands. Either way, the task/event gets stored in the expired list.
 
-### 2.7. Select a person : `select`
+**Marking a task or event as done**:
 
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+Format: `done INDEX`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+**Dismissing a task or event, without actually completing**:
+
+Format: `dis INDEX [\\REASON]`
+
+Example:
+
+`dis 2 \\dropped the module so no longer need to complete this assignment`
+
+### 3.6. Finding event or task by keyword: `find`
+
+> Finds tasks or events whose field(s) contain any of the given keywords.<br>
+
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
 Examples:
 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br>
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+* `find cs2103`<br>
+  Returns any task or event pertaining to cs2103
+
+### 3.7. Deleting a task/event without moving it to the expired list: `delete`
+
+> Just like edit, user can delete tasks and events in the active list by using their indexes specified in the most recent list. This option can also be used for tasks or events already in the expired list from which the user wants to get rid of.
+
+Format: `delete INDEX`
 
-### 2.8. Clearing all entries : `clear`
+### 3.8. Clearing all expired tasks and events : `clear`
 
-Clears all entries from the address book.<br>
 Format: `clear`
 
-### 2.9. Exiting the program : `exit`
+### 3.9 Viewing other material: `view`
 
-Exits the program.<br>
-Format: `exit`
+**View command history**: 
 
-### 2.10. Saving the data
+Format: `view history`
 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
-There is no need to save manually.
+**View the calendar**: 
 
-## 3. FAQ
+Format: `view calendar`
```

Comment:
i get a feeling, this is related to the UI navigation? if so, please specify that (or tell what is the purpose of `view` command)